src/Tizen.NUI/src/public/BaseComponents/TextEditorEvent.cs

```diff
@@ -371,6 +426,27 @@ public ScrollState ScrollState
             }
         }
 
+        /// <summary>
+        /// The CursorMoved event arguments.
+        /// </summary>
+        public class CursorMovedEventArgs : EventArgs
+        {
+            /// <summary>
+            /// TextEditor.
+            /// </summary>
+            public TextEditor TextEditor { get; set;}
```

Comment:
I think this property is not need, because sender will be TextEditor

Reply:
Remove TextEditor property and extract to public class (not nested) and then share with `TextField` class

Reply:
Done